.travis.yml

```diff
@@ -40,6 +38,14 @@ before_install:
   - wget https://www.dropbox.com/s/hhu6ojkgru93poh/frozen_unet_histology.pb?dl=1 -O data/frozen_unet_histology.pb
   # TODO: Implement frozen_unet_histology.pb model conversion with OpenVINO
   # (note that TensorFlow accepts input tensor of shape 1,952,1144,1,1,3)
+  - sudo apt-get install -y python3-pip
+  - sudo apt-get install -y python3-setuptools
+  - python3 -m pip install networkx --quiet
+  - python3 -m pip install defusedxml --quiet
+  - python3 -m pip install --upgrade pip --quiet
+  - python3 -m pip install tensorflow==1.15.0 numpy --quiet
```

Comment:
Please create `requirements.txt` file in 5_segmentation with the following content:
```
networkx
defusedxml
tensorflow==1.15.0
```
and call by 
```
python3 -m pip install -r modules/5_reinforcement_learning/requirements.txt --quiet
```